tests/cstor/snapshot/suite_test.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2019 The OpenEBS Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package snapshot
+
+import (
+	"flag"
+
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/tests"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	ns "github.com/openebs/maya/pkg/kubernetes/namespace/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	snapshot "github.com/openebs/maya/pkg/apis/openebs.io/snapshot/v1alpha1"
+	// auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var (
+	kubeConfigPath        string
+	openebsNamespace      = "openebs"
+	nsName                = "cstor-snap"
+	scName                = "cstor-test"
+	openebsCASConfigValue = "- name: ReplicaCount\n  value: 1\n- name: StoragePoolClaim\n  value: sparse-pool-auto"
+	openebsProvisioner    = "openebs.io/provisioner-iscsi"
+	spcName               = "sparse-pool-auto"
```

Comment:
Please refer to review comments at https://github.com/openebs/maya/pull/1215 for naming a cluster scoped object

Reply:
done